plugins/woocommerce/client/admin/client/settings-payments/settings-payments-main.tsx

```diff
@@ -31,10 +31,21 @@ export const SettingsPaymentsMain = () => {
 	);
 	const { installAndActivatePlugins } = useDispatch( PLUGINS_STORE_NAME );
 
+	// Get WooCommerce store country, split by colon to avoid the state.
+	const storeDefaultCountry =
+		window.wcSettings?.admin?.preloadSettings?.general?.woocommerce_default_country?.split(
+			':'
+		)[ 0 ];
+
 	const [ errorMessage, setErrorMessage ] = useState< string | null >( null );
 	const [ livePaymentsModalVisible, setLivePaymentsModalVisible ] =
 		useState( false );
 
+	const [ storeCountry, setStoreCountry ] = useState< string | null >(
+		window.wcSettings?.admin?.woocommerce_payments_nox_profile?.location ||
```

Comment:
This is not needed as we should fallback on the backend.

Reply:
But we still need to display it on the frontend, and we aren't getting the country from the backend, only a list of providers

Reply:
Now we do with these [two](https://github.com/woocommerce/woocommerce/pull/53505/commits/1040433148e0d570d08582bb043cff9e015b3c2a) [commits](https://github.com/woocommerce/woocommerce/pull/53505/commits/e90249c7d3f000d7eea0bf26458ff4342102dea0)

Reply:
Oh. I stand corrected. This is the only thing needed. But it is guaranteed to always be available, now.